pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StreamGitHubEventsCommand.java

```diff
@@ -36,10 +38,21 @@
   @CommandLine.Option(names = {"-personalAccessToken"}, required = true, description = "GitHub personal access token.")
   private String _personalAccessToken;
 
+  @CommandLine.Option(names = {"-sourceType"}, defaultValue = "Kafka",
+      description = "Stream DataSource to use for ingesting data. Supported values - Kafka,Kinesis")
+  private String _sourceType;
+
   @CommandLine.Option(names = {"-kafkaBrokerList"},
       description = "Kafka broker list of the kafka cluster to produce events.")
   private String _kafkaBrokerList = KafkaStarterUtils.DEFAULT_KAFKA_BROKER;
 
+  @CommandLine.Option(names = {"-kinesisEndpoint"},
```

Comment:
optional:
provide optional accessKey and secretKey too? one may want to test with a real kinesis and "access" and "secret" may not work rt? I see this command being very useful, and can see folks wanting to use this for their own setup tests
The `getKinesisStreamDataProducer` would also need to be updated accordingly.

Reply:
Those should not be provided as part of properties. The DefaultCredentialsProvider just picks them up from Env variables or ~/.aws/credentials file. It is a set standard by AWS.

Reply:
done.